tests/test_type_inference/test_compare.py

```diff
@@ -23,16 +22,10 @@ def test_compare_equality(left_value, operator_value_tuples):
     assert compare_node.inf_type.getValue() == bool
 
 
-@nottest
-@given(hs.lists(cs.comparator_operator, min_size=3), cs.homogeneous_list(min_size=4))
+@given(hs.lists(cs.comparator_operator, min_size=3), cs.numeric_list(min_size=4))
```

Comment:
Test previously failed due to attempt to apply '<' and '>' to booleans.

Reply:
This is a nice observation, thanks. Although I guess `bool` inherits from `int`, so eventually we'll allow this. :)